### PR TITLE
 make some existing logic and files easier to read

### PR DESCRIPTION
**Pull request by deads2k**
/hold

holding until I get library-go in

/assign @mfojtik

### CODE REVIEW

File: pkg/operator/sync_apiservice_v311_00.go [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+		errors = append(errors, fmt.Errorf("%q: %v", "svc", err))
+	}
+
+	requiredSA := resourceread.ReadServiceAccountV1OrDie(v310_00_assets.MustAsset("v3.10.0/apiservice-cabundle-controller/sa.yaml"))
+	_, saModified, err := resourceapply.ApplyServiceAccount(c.corev1Client, requiredSA)
+	if err != nil {
+		errors = append(errors, fmt.Errorf("%q: %v", "sa", err))
+	}
+
+	// TODO create a new configmap whenever the data value changes
```
Comment (mfojtik) [inline pkg/operator/sync_apiservice_v311_00.go]: @deads2k do we just mutate the config map today?

File: pkg/operator/sync_apiservice_v311_00.go [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+	// TODO check basic preconditions here
+	actualDeployment, _, err := manageAPIServiceDeployment_v311_00_to_latest(c.appsv1Client, operatorConfig, previousAvailability, forceDeployment)
+	if err != nil {
+		errors = append(errors, fmt.Errorf("%q: %v", "deployment", err))
+	}
+
+	return resourcemerge.ApplyGenerationAvailability(versionAvailability, actualDeployment, errors...), errors
+}
+
+func manageAPIServiceConfigMap(client coreclientv1.ConfigMapsGetter, operatorConfig *scsv1alpha1.ServiceCertSignerOperatorConfig) (*corev1.ConfigMap, bool, error) {
```
Comment (mfojtik) [inline pkg/operator/sync_apiservice_v311_00.go]: shouldn't this be manageAPIServiceDeployment_v310_00_to_latest ?

Comment (mfojtik) [reply pkg/operator/sync_apiservice_v311_00.go]: or "manageAPIServiceDeployment_v310_00_to_v311_00"

File: pkg/operator/sync_apiservice_v311_00.go (hunk 1 above)
Comment (deads2k) [reply pkg/operator/sync_apiservice_v311_00.go]: > @deads2k do we just mutate the config map today?

Yes.

**Comment by deads2k** [comment]
names fixed, tagging per slack

**Pull request merged**